contrib/linux/install-icons.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+DEST_PATH="$HOME/.local/share"
+REMOVE=false
+USAGE=false
+
+strstr() {
+    [ "${1#*$2*}" = "$1" ] && return 1
+    return 0
+}
+
+usage() {
+    echo "Installs icon files and desktop shortcut."
+    echo "--system: Install to system wide directory (/usr/share)"
+    echo "--remove: Remove icons and desktop file"
```

Comment:
desktop shortcut?

Reply:
It's the .desktop file.  It creates a shortcut to Dosbox in the "start menu" (idk what the proper name is in Linux, I just call it that since Windows 95).  Actual location can vary based on desktop environment.  This is also what tells the desktop environment to use the icons.  Tested in KDE that the icon shows even if you launch Dosbox from the terminal (as long as the .desktop file is present).

Reply:
Cool, I just noticed you're saying "desktop shortcut" in the first line, that's all 😄